include/ensmallen_bits/function/add_evaluate.hpp

```diff
@@ -60,7 +60,7 @@ class AddEvaluate<FunctionType,
     return static_cast<FunctionType*>(
         static_cast<Function<FunctionType,
                              MatType, GradType>*>(this))->Evaluate(coordinates);
-  }
+   }
```

Comment:
```suggestion
  }
```
I think that this was right before.